kernels/build.py

```diff
@@ -92,7 +92,7 @@ def build_kernel(source, output=None, flags=''):
 %xmm0,%r8d' 'movq %mm4,%r8' 'movq %r8,%mm5' 'movq %r8,%xmm0' 'movq %xmm0,%r8' 'pcmpestri $0x7e,0x100(%rsp),%xmm0' 'pcmpestri $0x7e,%xmm0,%xmm0' 'pcmpestri $0x7e,%xmm0,%xmm0' 'pcmpestrm $0x7e,0x100(%rsp),%xmm0' 'pcmpestrm $0x7e,%xmm0,%xmm0' 'pcmpistri $0x7e,0x100(%rsp),%xmm0' 'pcmpistri $0x7e,%xmm0,%xmm0' 'pcmpistrm $0x7e,0x100(%rsp),%xmm0' 'pcmpistrm $0x7e,%xmm0,%xmm0' 'pextrb $0x7e,%xmm0,%r8d' 'pextrd $0x7e,%xmm0,%r8d' 'pextrq $0x7e,%xmm0,%r8' 'pextrw $0x7e,%mm7,%r8d' 'pextrw $0x7e,%xmm0,%r8d' 'pinsrb $0x7e,%r8d,%xmm0' 'pinsrd $0x7e,%r8d,%xmm0' 'pinsrq $0x7e,%r8,%xmm0' 'pinsrw $0x7e,%r8d,%mm6' 'pinsrw $0x7e,%r8d,%xmm0' 'pmovmskb %mm0,%r8d' 'pmovmskb %xmm0,%r8d' 'ptest 0x100(%rsp),%xmm0' 'ptest %xmm0,%xmm0' 'ucomisd %xmm0,%xmm1' 'ucomiss %xmm0,%xmm0' 'ucomiss 0x100(%rsp),%xmm0' 'ucomisd 0x100(%rsp),%xmm0' 'vaeskeygenassist $0x7e,0x100(%rsp),%xmm0' 'vaeskeygenassist $0x7e,%xmm0,%xmm1' 'vcomiss 0x100(%rsp),%xmm0' 'vcomiss %xmm0,%xmm1' 'vcvtsd2si 0x100(%rsp),%r8' 'vcvtsd2si 0x100(%rsp),%r8d' 'vcvtsd2si %xmm0,%r8' 'vcvtsd2si %xmm0,%r8d' 'vcvtsd2usi 0x100(%rsp),%r8' 'vcvtsd2usi %xmm0,%r8d' 'vcvtsi2sd %r8,%xmm0,%xmm1' 'vcvtsi2sd %r8d,%xmm0,%xmm1' 'vcvtsi2ss %r8d,%xmm0,%xmm1' 'vcvtsi2ss %r8d,%xmm0,%xmm1' 'vcvtsi2ss %rsi,%xmm0,%xmm1' 'vcvtss2si 0x100(%rsp),%r8' 'vcvtss2si 0x100(%rsp),%r8d' 'vcvtss2si %xmm0,%r8' 'vcvtss2si %xmm0,%r8d' 'vcvtss2usi 0x100(%rsp),%r8' 'vcvtss2usi 0x100(%rsp),%r8d' 'vcvtss2usi %xmm0,%r8' 'vcvtss2usi %xmm0,%r8d' 'vcvttsd2si 0x100(%rsp),%r8' 'vcvttsd2si 0x100(%rsp),%r8d' 'vcvttsd2si %xmm0,%r8' 'vcvttsd2si %xmm0,%r8d' 'vcvttsd2usi 0x100(%rsp),%r8d' 'vcvttsd2usi %xmm0,%r8' 'vcvttss2si 0x100(%rsp),%r8' 'vcvttss2si 0x100(%rsp),%r8d' 'vcvttss2si %xmm0,%r8' 'vcvttss2si %xmm0,%r8d' 'vcvttss2usi 0x100(%rsp),%r8' 'vcvttss2usi %xmm0,%r8' 'vcvtusi2sd %r8d,%xmm0,%xmm1' 'vcvtusi2ss %r8d,%xmm0,%xmm1' 'vcvtusi2ss %r8,%xmm0,%xmm1' 'vextractps $0x7e,%xmm0,%r8d' 'vmovd %r8d,%xmm0' 'vmovd %xmm0,%r8' 'vmovq %xmm0,%r8' 'vmovmskpd %xmm0,%r8d' 'vmovmskpd %ymm0,%r8d' 'vmovmskps %xmm0,%r8' 'vmovmskps %ymm0,%r8' 'vmovq %r8,%xmm0' 'vpcmpestri $0x7e,0x100(%rsp),%xmm0' 'vpcmpestri $0x7e,%xmm0,%xmm1' 'vpcmpestrm $0x7e,0x100(%rsp),%xmm0' 'vpcmpestrm $0x7e,%xmm0,%xmm1' 'vpcmpistri $0x7e,0x100(%rsp),%xmm0' 'vpcmpistri $0x7e,%xmm0,%xmm0' 'vpcmpistrm $0x7e,0x100(%rsp),%xmm0' 'vpcmpistrm $0x7e,%xmm0,%xmm0' 'vpextrb $0x7e,%xmm0,%r8d' 'vpextrd $0x7e,%xmm0,%r8d' 'vpextrq $0x7e,%xmm0,%r8' 'vpextrw $0x7e,%xmm0,%r8d' 'vpinsrb $0x7e,%r8d,%xmm0,%xmm1' 'vpinsrd $0x7e,%r8d,%xmm0,%xmm1' 'vpinsrq $0x7e,%r8,%xmm0,%xmm1' 'vpinsrw $0x7e,%r8d,%xmm0,%xmm1' 'vpmovmskb %xmm0,%r8d' 'vpmovmskb %ymm0,%r8d' 'vptest 0x100(%rsp),%xmm0' 'vptest 0x100(%rsp),%ymm0' 'vptest %xmm0,%xmm0' 'vptest %ymm0,%ymm0' 'vtestpd 0x100(%rsp),%xmm0' 'vtestpd 0x100(%rsp),%ymm0' 'vtestpd %xmm0,%xmm0' 'vtestpd %ymm0,%ymm0' 'vtestps 0x100(%rsp),%xmm0' 'vtestps 0x100(%rsp),%ymm0' 'vtestps %xmm0,%xmm0' 'vtestps %ymm0,%ymm0' 'vucomisd 0x100(%rsp),%xmm0' 'vucomisd %xmm0,%xmm0' 'vucomiss %xmm0,%xmm1' -n1 ",'v2ii2v')
```

Comment:
Why did you put all the cases in? Is it a must? I believe a simple kernel with 1-2 cases is enough, just to ensure new stats are working when needed.

Reply:
reduced testcases